CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ if(MSVC)
     string(REGEX REPLACE "/W3" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
     # we would probably like to pass /Wall instead of /W4, but there's a lot
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /permissive- /W4")
+    #LibPNG is not linking on MSVC correctly, disable it.
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DMIN_BUILD")
```

Comment:
Please do not. see also https://github.com/omf2097/openomf/pull/769

Reply:
This was temporary you can take it out when 769 is merged. Or I'll fix it if 769 goes in first.